server/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "rate-my-diy",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "server": "nodemon index.js"
```

Comment:
In the future, we will need one command that runs server with node instead of nodemon because it's a dev dependency.